CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # App Center SDK for .NET Change Log
 
+## Version 1.11.0
+
+### AppCenter
+
+### iOS
+
+* **[Fix]** Fix an issue where concurrent modification of custom properties was not thread safe.
+* **[Fix]** Fix validating and discarding Not a Number (NaN) and infinite double values for custom properties.
+* **[Fix]** Use standard SQL syntax to avoid affecting users with custom SQLite libraries.
+* **[Fix]** Get database page size dynamically to support custom values.
+
+### AppCenterCrashes
+
+### Android
+
+* **[Fix]** Preventing stack overflow crash while reading a huge throwable file.
+
+### iOS
```

Comment:
Please remove the iOS section for Crashes as we don't support mac os for Xamarin.